UnityProject/Assets/Scripts/UI/Core/Net/Elements/NetColorChanger.cs

```diff
@@ -45,13 +45,13 @@ public class NetColorChanger : NetUIElement<Color>
 
 		public Graphic Element => element ??= GetComponent<Graphic>();
 		private Graphic element;
-		
+
 		public override void ExecuteServer(PlayerInfo subject) { }
 
 		/// <summary>
 		/// Server-only method for updating element (i.e. changing label text) from server GUI code
 		/// </summary>
-		public override void SetValueServer(Color value)
```

Comment:
"set value server" makes much more sense than MasterSetValue, why the rename?

Reply:
because when you host the game yourself there is technically two UIs, one is the one that is considered server/master
and then there's a client immolated version one as well, so you can be on the server while still being on the client one